pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -544,37 +547,72 @@ func (qjm *XController) GetAggregatedResourcesPerGenericItem(cqj *arbv1.AppWrapp
 	// Get all pods and related resources
 	for _, genericItem := range cqj.Spec.AggrResources.GenericItems {
 		itemsList, _ := genericresource.GetListOfPodResourcesFromOneGenericItem(&genericItem)
-		for i :=0; i < len(itemsList); i++ {
+		for i := 0; i < len(itemsList); i++ {
 			retVal = append(retVal, itemsList[i])
 		}
 	}
 
 	return retVal
 }
+
+func (qjm *XController) GetAllObjectsOwned(cqj *arbv1.AppWrapper) {
```

Comment:
The `QJ` references are slowly being updated to `AW` as code is modified.  Could you update your new functions from `qj` to `aw` or in this case `caw`?  Thanks!

Reply:
Done